### PR TITLE
Styling Gubernator PR dashboard

### PR DESCRIPTION
**Pull request by qhuynh96**
g02pzjvd2ge
screen shot 2018-02-13 at 1 43 50 am 1
screen shot 2018-02-13 at 1 33 59 am

### CODE REVIEW

**Comment by BenTheElder** [comment]
❄️ ?
/retest

**Comment by stevekuznetsov** [comment]
Bona fide `gubernator` test failure:

```
W0213 19:45:19.960] FAIL: Individual PR pages are case insensitive.
W0213 19:45:19.960] ----------------------------------------------------------------------
W0213 19:45:19.961] Traceback (most recent call last):
W0213 19:45:19.961]   File "/workspace/k8s.io/test-infra/gubernator/view_pr_test.py", line 171, in test_case_insensitive
W0213 19:45:19.961]     self.assertIn('Needs Attention (1)', resp)
W0213 19:45:19.961] AssertionError: 'Needs Attention (1)' not found in <200 OK text/html body='<!DOCTYPE...tml>'/6814>
```

File: gubernator/static/style.css [hunk 1]
```diff
@@ -1,25 +1,29 @@
 body {
-    font-family: sans-serif;
+    font-family: 'Roboto', sans-serif !important;
```
Comment (stevekuznetsov) [inline gubernator/static/style.css]: Why `!important`?

Comment (BenTheElder) [reply gubernator/static/style.css]: because the mdl stylesheet also sets body { font-family: ... }

**Comment by rmmh** [comment]
I'll have a look at this on Thursday, thanks!

**Comment by rmmh** [comment]
Making all text smaller makes legibility worse. Invisible horizontal scrollbars for windows under 1500px is bad usability. The ACK buttons don't look like clickable targets.

https://k8s-gubernator.appspot.com/pr/rmmh vs 
https://test-dot-k8s-gubernator.appspot.com/pr/rmmh

**Comment by qhuynh96** [comment]
@rmmh comments resolved

File: gubernator/static/style.css [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
     padding: 0;
     margin: 0;
     scroll-behavior: smooth;
 }
 #header {
+    align-items: center;
```
Comment (rmmh) [inline gubernator/static/style.css]: Please don't change this. this breaks the build page styling and is irrelevant for this PR.

screen shot 2018-02-16 at 1 23 09 pm

File: gubernator/static/style.css [hunk 3]
```diff
@@ -206,19 +201,158 @@ div#log {
...
+.line-break {
+    background-color: #000000;
+    border: none;
+    color: #000000;
+    height: 2px;
+    margin: 24px 0 8px 0;
+    opacity: 0.08;
+}
+
+.db-assignee {
```
Comment (rmmh) [inline gubernator/static/style.css]: why is this bold? (why are "status" and "assignee" bold vs "title", etc.)

File: gubernator/templates/pr_dashboard.html [hunk 4]
```diff
@@ -9,62 +13,97 @@
...
-		% endfor
-	</table>
+    <table class="pr-dashboard-table mdl-data-table mdl-js-data-table mdl-shadow--1dp">
+        <thead>
+            <tr>
+                <th class="db-repo mdl-data-table__cell--non-numeric">Repo</th>
+                <th class="db-number">Number</th>
+                <th class="db-author mdl-data-table__cell--non-numeric">Author</th>
+                <th class="db-status mdl-data-table__cell--non-numeric">Status</th>
+                <th class="db-update mdl-data-table__cell--non-numeric">Updated</th>
```
Comment (rmmh) [inline gubernator/templates/pr_dashboard.html]: needs to include `&#9660;` to indicate sort direction

**Comment by qhuynh96** [comment]
@rmmh Do you want these changes to Gubernator? If yes can continue to fix the styles, otherwise I can drop this

**Comment by fejta** [comment]
/lifecycle stale
/assign @michelle192837 @rmmh 

@qhuynh96 any plans to work on this? Pretty please! :cake: 
If not does someone want to take over this PR?

**Comment by fejta** [comment]
Any signs of life on this PR? Would definitely like to see these changes! Please close and/or rebase.

**Comment by qhuynh96** [comment]
/remove-lifecycle stale
I am addressing previous comments

**Comment by qhuynh96** [comment]
@fejta @rmmh I cleaned up the PR and resolve previous comments. Ready for another review round

**Comment by qhuynh96** [comment]
any feedback for this @stevekuznetsov @rmmh ?

**Comment by stevekuznetsov** [comment]
@qhuynh96 do you have screenshots of the current iteration?

**Pull request closed without merging**